scripts/Tools/check_case

```diff
@@ -50,9 +50,8 @@ def _main_func(description):
 
     parse_command_line(sys.argv, description)
 
-    check_lockedfiles()
-
     with Case(read_only=False) as case:
+        check_lockedfiles(case)
```

Comment:
Why has the check_lockedfiles been moved here with a new argument? Was this tied to multiple couplers - or was this needed regardless?